app/src/main/res/values/strings_settings.xml

```diff
@@ -63,6 +64,8 @@
     <string name="settings_desc_special_features">Uncheck to disable the special features again</string>
     <string name="settings_desc_enable_screenshot">Allow to take screenshots of the main screen
         (disabled by default for security reasons)</string>
+    <string name="settings_desc_enable_android_backup_service">Enables andOTP to use android\'s
+        built in backup service to bacup keys and preferences</string>
```

Comment:
there's a small typo (`bac[k]up keys`)

Reply:
Just fixed it in 263a2097ce2ac8a71b4a41798dad5640df0f77a3, thanks for noticing.

Reply:
Well, thank you for your work on andOTP :wink: